src/python/grpcio/grpc/_channel.py

```diff
@@ -1312,8 +1447,16 @@ def create(flags, method, host, deadline, metadata, credentials,
 
 
 class _ChannelConnectivityState(object):
-
-    def __init__(self, channel):
+    lock: threading.RLock
+    channel: grpc.Channel
+    polling: bool
+    connectivity: grpc.ChannelConnectivity
+    try_to_connect: bool
+    callbacks_and_connectivities: Sequence[Sequence[Union[
```

Comment:
I think the outer container should be `List` here. The inner container is more difficult. It looks like it's currently a list of fixed size 2 where the zeroth element is of type `Callable[[grpc.ChannelConnectivity], None]` and first element is of type `Optional[grpc.ChannelConnectivity]`. Can we just change the sites that mutate this member to use tuples so that this type can become `List[Tuple[Callable[[grpc.ChannelConnectivity], None], Optional[grpc.ChannelConnectivity]]]`?

Reply:
Please create a github issue and add a TODO.

Reply:
Sure, issue: https://github.com/grpc/grpc/issues/31704